### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by JimboJoe**
## Problem
- *Upgrade from previous commit fails because acl dependency is missing.*
- *Fix https://github.com/YunoHost-Apps/transmission_ynh/issues/47*

## Solution
- *Use the commit with this fix.*
- *Add the missing options*

## PR Status
- [x] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : frju365 
- [x] **Approval (LGTM)** : frju365
- [x] **Approval (LGTM)** : Kay0u
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/transmission_ynh%20testing%20(Official)/)

### CODE REVIEW

**Comment by JimboJoe** [comment]
Now ready for review.

File: conf/settings.json [hunk 1]
```diff
@@ -46,6 +46,8 @@
     "rpc-authentication-required": false, 
     "rpc-bind-address": "127.0.0.1", 
     "rpc-enabled": true, 
+    "rpc-host-whitelist": "", 
+    "rpc-host-whitelist-enabled": true,
```
Comment (kay0u) [inline conf/settings.json]: I don't understand, why set it to true when in #47 the fix proposed is to set it to false? Or maybe add \_\_DOMAIN\_\_ in the line above?

Comment (maniackcrudelis) [reply conf/settings.json]: Looks like the fix, https://github.com/YunoHost-Apps/transmission_ynh/pull/48 was wrong, and no one never noticed it...

Comment (maniackcrudelis) [reply conf/settings.json]: Both with false or true, it works on my server.

**Comment by JimboJoe** [comment]
Will be released in 3 days.

**Pull request merged**